roles/pulp_devel/defaults/main.yml

```diff
@@ -2,3 +2,4 @@
 pulp_requirements_dir: "{{ pulp_source_dir }}"
 pulp_devel_package_retries: 5
 pulp_devel_supplement_bashrc: false
+pulp_workers: 2
```

Comment:
nitpick: can this have a newline at the end of it. If someone were to concat this with another yml file it would break without it.